src/DataBus/NServiceBus.DataBus.AzureBlobStorage.csproj

```diff
@@ -17,14 +17,15 @@
     <UpdateAssemblyInfo>true</UpdateAssemblyInfo>
     <NoWarn>1591, 0618</NoWarn>
```

Comment:
I've ran into this with ASQ when nowarns where disabling comments on public types (CS1591) - we should get rid of it and probably provide xmldoc.
CS0618 should not be needed either.

Reply:
That is fine, but I suggest we address it in a separate PR.